code/datums/materials/_material.dm

```diff
@@ -78,6 +78,30 @@ Simple datum which is instanced once per type and is used for every object of sa
 	/// The slowdown that is added to items.
 	var/added_slowdown = 0
 
+	/// Fish made of or infused with this material have their weight multiplied by this value.
+	var/fish_weight_modifier = 1
+
+	/// Fishing rods made of this material have an extra chance of catching fish made of the main mat the rod is made of.
+	var/material_fish_extra_chance = 0
+	/// If true, fishing rods made of this material will attract shiny lover fish. Ditto with fishing equipment
+	var/is_shiny_fishing_lure = FALSE
+	/// Additive bonus/malus to the fishing difficulty modifier of any rod made of this item. Negative is good, positive bad
+	var/fishing_difficulty_modifier = 0
+	/// Additive bonus/malus to the cast range of the fishing rod
+	var/fishing_cast_range = 0
+	/// The multiplier of how much experience is gained when using a fishing rod made of this material
+	var/fishing_experience_multiplier = 1
+	/// The multiplier to the completion gain of the fishing rod made of this material
+	var/fishing_completion_speed = 1
+	/// The multiplier of the bait/bobber speed of the fishing challenge for fishing rods made of this material
+	var/fishing_bait_speed_mult = 1
+	/// The multiplier of the deceleration/friction for fishing rods made of this material
+	var/fishing_deceleration_mult = 1
+	/// The multiplier of the bounciness of the bait/bobber upon hitting the edges of the minigame area
+	var/fishing_bounciness_mult = 1
+	/// The multiplier of negative velocity that pulls the bait/bobber of a fishing rod down when not holding the click
+	var/fishing_gravity_mult = 1
```

Comment:
Reposting from discord, this is half the length of our current vars if not more - you need to cut down on these, perhaps replace some of them with existing material properties